Cargo.lock

```diff
@@ -952,18 +952,18 @@ checksum = "92d43fe69e652f3df9bdc2b85b2854a0825b86e4fb76bc44d945137d053639ca"
 
 [[package]]
 name = "serde"
-version = "1.0.198"
+version = "1.0.199"
```

Comment:
**Security Issue**: The version update for serde introduces a new checksum which needs to be verified to ensure it has not been tampered with or incorrectly generated. <br> **Fix**: Verify the new checksum '0c9f6e76df036c77cd94996771fb40db98187f096dd0b9af39c6c6e452ba966a' for serde version 1.0.199 with the source or through a secure, trusted mechanism. <br> **Code Suggestion**: 
 ```
 -checksum = "9846a40c979031340571da2545a4e5b7c4163bdae79b301d5f86d03979451fcc"
 +checksum = "0c9f6e76df036c77cd94996771fb40db98187f096dd0b9af39c6c6e452ba966a"
 ```